### PR TITLE
Modified style.css,index.html and index.js file

### PR DESCRIPTION
**Pull request by vk20july**
Mam Please Review my code

### CODE REVIEW

File: js_files/index.js [hunk 1]
```diff
@@ -4,10 +4,23 @@ document.addEventListener("keyup", (e) => {
...
-  cont.style.display = cont.style.display == "none" ? "block" : "none";
-}
+const calender=document.getElementById("cont");
+let calender1=window.getComputedStyle(calender).getPropertyValue("display");
+const log=document.getElementById("btn1");
+
+console.log(calender1);
+console.log(log);
+log.addEventListener("click",(e)=>{
+  console.log(calender1);
```
Comment (devvsakib) [inline js_files/index.js]: delete all console log code from the file. It's not a good practice to put console code

**Review by devvsakib — CHANGES_REQUESTED** [review]
also replace let/const instead var.

**Comment by vk20july** [comment]
ok sir

**Comment by vk20july** [comment]
@devvsakib sir made the changes please review

**Review by devvsakib — CHANGES_REQUESTED** [review]
I actually said to use let or const without using var. I didn't mean to use var variable.

Line 8 use const.
Lines 24 to 31 replace var with let.

**Comment by nisha331** [comment]
@vk20july can i have the screenrecoding of the task.. So i can see if things are working fine

**Comment by devvsakib** [comment]
> @vk20july can i have the screenrecoding of the task.. So i can see if things are working fine

you can add vercel to this project, so when someone makes PR, vercel will automatically deploy your site, and give a preview before merging.

**Comment by nisha331** [comment]
> > @vk20july can i have the screenrecoding of the task.. So i can see if things are working fine
> 
> you can add vercel to this project, so when someone makes PR, vercel will automatically deploy your site, and give a preview before merging.

yeah will do  that for the moment he can just give the screenrecording

**Comment by vk20july** [comment]
Ok @nisha331 @devvsakib will create proper pull request by tonight

**Comment by vk20july** [comment]
https://user-images.githubusercontent.com/71543332/220919966-60bd09e4-3139-4987-bca6-b652245304b3.mp4

**Comment by vk20july** [comment]
@nisha331 @devvsakib kindly review the changes

**Review by nisha331 — APPROVED** [review]

**Pull request merged**